### PR TITLE
Add examples for bitbucketCloud actions

### PR DESCRIPTION
**Pull request by BillyMichael**
#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [] Added or updated documentation
- [x] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

#### Screenshot

Screenshot 2024-03-25 at 18 08 27

### CODE REVIEW

File: plugins/scaffolder-backend-module-bitbucket-cloud/src/actions/bitbucketCloud.examples.ts [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+          input: {
+            repoUrl:
+              'bitbucket.org?repo=repo&workspace=workspace&project=project',
+          },
+        },
+      ],
+    }),
+  },
+  {
+    description: 'Add a description.',
```
Comment (punkle) [inline plugins/scaffolder-backend-module-bitbucket-cloud/src/actions/bitbucketCloud.examples.ts:39]: Should this say "Initializes a git repository in Bitbucket cloud and sets a description."

File: plugins/scaffolder-backend-module-bitbucket-cloud/src/actions/bitbucketCloud.examples.ts [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+            repoUrl:
+              'bitbucket.org?repo=repo&workspace=workspace&project=project',
+            description: 'Initialize a git repository',
+          },
+        },
+      ],
+    }),
+  },
+  {
+    description: 'Change visibility of the repository.',
```
Comment (punkle) [inline plugins/scaffolder-backend-module-bitbucket-cloud/src/actions/bitbucketCloud.examples.ts:56]: Initializes a new public repository in Bitbucket.

File: plugins/scaffolder-backend-module-bitbucket-cloud/src/actions/bitbucketCloud.examples.ts [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+              'bitbucket.org?repo=repo&workspace=workspace&project=project',
+            description: 'Initialize a git repository',
+            repoVisibility: 'public',
+          },
+        },
+      ],
+    }),
+  },
+  {
+    description: 'Set the default branch.',
```
Comment (punkle) [inline plugins/scaffolder-backend-module-bitbucket-cloud/src/actions/bitbucketCloud.examples.ts:74]: Create a new public Bitbucket repository along with a default branch called `main`.

File: plugins/scaffolder-backend-module-bitbucket-cloud/src/actions/bitbucketCloud.examples.ts [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+          id: 'publish',
+          action: 'publish:bitbucketCloud',
+          name: 'Publish to Bitbucket Cloud',
+          input: {
+            repoUrl:
+              'bitbucket.org?repo=repo&workspace=workspace&project=project',
+            description: 'Initialize a git repository',
+            repoVisibility: 'public',
+            defaultBranch: 'main',
+            token: 'your-custom-auth-token',
```
Comment (punkle) [inline plugins/scaffolder-backend-module-bitbucket-cloud/src/actions/bitbucketCloud.examples.ts:126]: Perhaps it makes sense to include the repo url picker parameter definition in the example in order to fill out this example a little bit. Thoughts?

File: plugins/scaffolder-backend-module-bitbucket-cloud/src/actions/bitbucketCloud.examples.ts (hunk 1 above)
Comment (BillyMichael) [reply plugins/scaffolder-backend-module-bitbucket-cloud/src/actions/bitbucketCloud.examples.ts:39]: Happy to make that change. Was following a similar convention to the GitHub action examples but agree your naming makes more sense.

Comment (punkle) [reply plugins/scaffolder-backend-module-bitbucket-cloud/src/actions/bitbucketCloud.examples.ts:39]: Yeah, I know what you mean. This example is definitely better than having no examples. So i would approve in any case. :)

**Review by punkle — APPROVED** [review]

**Review by Rugvip — APPROVED** [review]
Thank you! 👍 

Let's :shipit: 🎉

**Pull request merged**